src/test/java/hudson/tasks/MailerTest.java

```diff
@@ -37,12 +40,21 @@
 
 import jenkins.model.JenkinsLocationConfiguration;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertNull;
+
 
 /**
  * @author Kohsuke Kawaguchi
  */
-public class MailerTest extends HudsonTestCase {
+public class MailerTest {
+
+    @Rule
+    public JenkinsRule rule = new JenkinsRule();
+
     @Bug(1566)
```

Comment:
When migrating from `Bug` to `Issue` is some "marker" used to "decorate" the previous value or the plain number represents the old bug tracking system?